### PR TITLE
Lens/fake smart auth

### PR DESCRIPTION
**Pull request by fanbomeng97**
- Fake SMART auth process, availiable PIN: **03825, 03826, 03827**, different PIN will display different patients, (Still can not get p104, p105, p106 ... )

- Add logo into scene

- Add welcome title into scene

### CODE REVIEW

**Comment by nbckr** [comment]
Hey, what do you mean when you say "Still can not get p104, p105, p106"? Is this something @boonwj can help you with?

**Comment by nbckr** [comment]
I have the Accessor running and it seems to work (200 response accoding to the log)
image

When I do the same query in the browser, I get:
image


But:
image

and:
image

**Comment by fanbomeng97** [comment]
@nbckr it seems like the birthdate of p100 is missing, so it will have this error, i will add an error handler

**Comment by fanbomeng97** [comment]
image

**Comment by nbckr** [comment]
Okay yeah. It would be good if you can handle most errors gracefully and just show "Unknown" if a field like birthDate is missing.

**Comment by nbckr** [comment]
Okay now I see patients. When I want to open a hologram however, there is a 500 error in the Accessor. I am using the current `dev` branch version. Don't you get the same?

image

```
REST Error: 500
{"errorCode":"500","errorMessage":"-\u003e github.com/Azure/azure-storage-blob-go/azblob.newStorageError, C:/Users/niels/go/src/github.com/Azure/azure-storage-blob-go/azblob/zc_storage_error.go:42\n===== RESPONSE ERROR (ServiceCode=BlobNotFound) =====\nDescription=The specified blob does not exist.\nRequestId:77aac71f-b01e-0073-4035-59737f000000\nTime:2019-08-22T22:02:19.4842942Z, Details: \n   Code: BlobNotFound\n   GET https://holoblob.blob.core.windows.net/holograms/h103?timeout=61\n   Authorization: REDACTED\n   User-Agent: [Azure-Storage/0.7 (go1.12.7; Windows_NT)]\n   X-Ms-Client-Request-Id: [e81c38dc-e0d3-4e44-6adb-ad79262cede2]\n   X-Ms-Date: [Thu, 22 Aug 2019 22:02:17 GMT]\n   X-Ms-Version: [2018-11-09]\n   --------------------------------------------------------------------------------\n   RESPONSE Status: 404 The specified blob does not exist.\n   Content-Length: [215]\n   Content-Type: [application/xml]\n   Date: [Thu, 22 Aug 2019 22:02:19 GMT]\n   Server: [Windows-Azure-Blob/1.0 Microsoft-HTTPAPI/2.0]\n   X-Ms-Error-Code: [BlobNotFound]\n   X-Ms-Request-Id: [77aac71f-b01e-0073-4035-59737f000000]\n   X-Ms-Version: [2018-11-09]\n\n\n"}
```

My settings are
```
export AZURE_STORAGE_ACCOUNT=holoblob
export AZURE_STORAGE_ACCESS_KEY=ri8F+lX6vu3i5FfMnjhz0uP8SvuYEgY26Gd8ZPYKyihue4a9ylj6Y7RsehrwFhlfZCWNgbqkqzFbN0TSPowVRg==
export ACCESSOR_FHIR_URL=https://holostorage.azurewebsites.net
export ENABLE_CORS=true
```

File: HoloRepositoryDemoApplication/Assets/Scripts/LoginAuthentication.cs [hunk 1]
```diff
@@ -20,26 +18,28 @@ private void Start()
...
         {
-            if (PIN.text == "3825")
-            {
-                Destroy(PINPad);
-                SceneManager.LoadScene("PatientListScene", LoadSceneMode.Additive);
-                SceneManager.LoadScene("HologramListScene", LoadSceneMode.Additive);
-            }
-            else
-            {
+            case "03825":
```
Comment (nbckr) [inline HoloRepositoryDemoApplication/Assets/Scripts/LoginAuthentication.cs]: Just like in UI please have a comment as to why these are hard-coded

File: HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Scripts/HoloStorageClient.cs [hunk 2]
```diff
@@ -291,16 +291,16 @@ public static Patient JsonToPatient(JSONNode json, string id)
 
             try
             {
-                patient.pid = json["pid"].Value;
-                patient.gender = json["gender"].Value;
-                patient.birthDate = json["birthDate"].Value;
+                patient.pid = json["pid"].Value == "" ? "Unknown" : json["pid"].Value;
```
Comment (nbckr) [inline HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Scripts/HoloStorageClient.cs]: This file is getting a bit long now, maybe you can extract the mapping/data extraction to a different class?

**Comment by fanbomeng97** [comment]
@nbckr  I am using the current `MISC/holo_demo_worldbranch` version, and my settings are:
```
AZURE_STORAGE_ACCOUNT=holoblobdemo

AZURE_STORAGE_ACCESS_KEY=JArFyfbP2EiICcvRBWgCClrPgntWdcHPb3wTXq35lQ1OdorIFa0dKrJS5qvnwZOGs0CEi02x6TZIwoXpWrWRZw==

ACCESSOR_FHIR_URL=https://holostorage-demo.azurewebsites.net
ENABLE_CORS=true
```
it works fine on my pc

**Comment by nbckr** [comment]
Okay. That is important information for me as the reviewer ;) With these settings, it works for me too. Cool then.

Btw maybe an empty patient should be a warning rather than an error log statement, as it is currently expected behaviour. (I would not change the hard-coded PIDs, cause now when I upload new data for these patients, they will show up on your end in the future)

**Pull request merged**